src/azure/cli/_azure_env.py

```diff
@@ -5,9 +5,7 @@
 
 CLIENT_ID = '04b07795-8ddb-461a-bbee-02f9e1bf7b46'
 
-ENV_DEFAULT = 'AzureCloud'
-ENV_US_GOVERNMENT = 'AzureUSGovernment'
-ENV_CHINA = 'AzureChinaCloud'
+ENV_DEFAULT = 'dogfood'
```

Comment:
The CLI does not support adding/switching environments yet and ENV_DEFAULT is used in multiple places directly. We will need to update all the places that use ENV_DEFAULT if we add one more. This is basically a temporary solution while involving as few lines of code as possible.